.codecov.yml

```diff
@@ -0,0 +1,20 @@
+coverage:
```

Comment:
Are we adding https://codecov.io/ build with this file?

Reply:
@mrizzi This file configures the rules for coverage reports into [https://codecov.io/](https://codecov.io/). Koku-ui also uses codecov for test coverage so I thought it might worth adding coverate to the current proyect. This is the coverage report of koku-ui https://codecov.io/gh/project-koku/koku-ui/

Reply:
Thanks, it seems something we could adopt as well so good to have it here.